clevertap-core/src/test/java/com/clevertap/android/sdk/CTWebInterfaceTest.kt

```diff
@@ -39,6 +39,43 @@ class CTWebInterfaceTest : BaseTestCase() {
 
     }
 
+
+    @Test
+    fun test_incrementValue_when_CalledWithKeyAndValue_should_CallAssocClevertapApiFunction() {
+        // when ctApi is null, calling this function will do nothing.
+        ctApi = null
+        ctWebInterface = CTWebInterface(ctApi)
+        ctWebInterface.incrementValue("key",1.5)
+
+
+        // when ctApi is not null, calling this function will call ctApi's internal function
+        ctApi = CleverTapAPI.getDefaultInstance(application)
+        val ctMock = Mockito.mock(CleverTapAPI::class.java)
+        ctWebInterface = CTWebInterface(ctMock)
+        ctWebInterface.incrementValue("key2",2.5)
+        Mockito.verify(ctMock,Mockito.times(1))?.incrementValue("key2",2.5)
+    }
```

Comment:
add tests for extreme ranges Int.MAX_VALUE , Double.MIN_VALUE, etc (for int/long/double/float)